governance/polls/Vault Compensation Poll 3

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Title of previous poll was: Proposal - Vault Compensation Plan Approval Poll - July 13, 2020.md

So title for this poll should probably be the same, with updated date. Note that the filename needs to end in '.md'